tests/bluetooth/tester/src/btp_bap.c

```diff
@@ -0,0 +1,206 @@
+/* btp_bap.c - Bluetooth BAP Tester */
+
+/*
+ * Copyright (c) 2021-2022 Nordic Semiconductor ASA
+ * Copyright (c) 2023 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/types.h>
+#include <stddef.h>
+#include <zephyr/kernel.h>
+
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/audio/audio.h>
+#include <zephyr/bluetooth/audio/pacs.h>
+
+#include "endpoint.h"
+#include "btp/btp.h"
+
+#define CONTROLLER_INDEX 0
+
+#define SUPPORTED_SINK_CONTEXT	(BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED | \
+				 BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL | \
+				 BT_AUDIO_CONTEXT_TYPE_MEDIA | \
+				 BT_AUDIO_CONTEXT_TYPE_GAME | \
+				 BT_AUDIO_CONTEXT_TYPE_INSTRUCTIONAL)
+
+#define SUPPORTED_SOURCE_CONTEXT (BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED | \
+				  BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL | \
+				  BT_AUDIO_CONTEXT_TYPE_MEDIA | \
+				  BT_AUDIO_CONTEXT_TYPE_GAME)
+
+#define AVAILABLE_SINK_CONTEXT SUPPORTED_SINK_CONTEXT
+#define AVAILABLE_SOURCE_CONTEXT SUPPORTED_SOURCE_CONTEXT
+
+static struct bt_codec lc3_codec =
+	BT_CODEC_LC3(BT_CODEC_LC3_FREQ_ANY, BT_CODEC_LC3_DURATION_10,
+		     BT_CODEC_LC3_CHAN_COUNT_SUPPORT(1),
+		     40u, 120u, 1u,
+		     (BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL |
+		     BT_AUDIO_CONTEXT_TYPE_MEDIA));
+
+static struct bt_pacs_cap cap_sink = {
+	.codec = &lc3_codec,
+};
+
+static struct bt_pacs_cap cap_source = {
+	.codec = &lc3_codec,
+};
+
+static int set_location(void)
+{
+	int err;
+
+	err = bt_pacs_set_location(BT_AUDIO_DIR_SINK,
+				   BT_AUDIO_LOCATION_FRONT_CENTER);
+	if (err != 0)
```

Comment:
hmm I think ifs should now always be inside { }  (not sure why CI style check doesn't catch that)

Reply:
I've deleted them after zephyrs checker `exec git diff HEAD~1 | ${ZEPHYR_BASE}/scripts/checkpatch.pl -` gave me:
```
WARNING: braces {} are not necessary for single statement blocks
#198: FILE: tests/bluetooth/tester/src/btp_bap.c:58:
+       if (err != 0) {
+               return err;
+       }
```

Reply:
The style guide only requires braces if there is multiple statements: 

https://www.kernel.org/doc/html/v4.10/process/coding-style.html#placing-braces-and-spaces

That being said, it's more common to always use braces in Zephyr, and the warning you see shouldn't appear in CI. Not sure why you get - Are you running `checkpath.pl` directly instead of the `check_compliance.py` script? 

In any case, I suggest to use braces in all case.

Reply:
Yes, I used the `checkpath.pl`. The `check_compliance.py` runs it too, but somehow skips this warning. So I will use the `check_compliance.py` next time.

Reply:
Just running `checkpath.pl` directly would not use any of the Zephyr configurations for the script, so that's why you are getting that warning :) 

Please use `check_compliancy.py` going forward